marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/entities/search/FacetHandler.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2012-2020 MarkLogic Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package com.marklogic.hub.central.entities.search;
+
+import com.marklogic.client.query.StructuredQueryBuilder;
+import com.marklogic.client.query.StructuredQueryDefinition;
+import com.marklogic.hub.central.entities.search.models.DocSearchQueryInfo;
+
+public interface FacetHandler {
+    String COLLECTION_CONSTRAINT_NAME = "Collection";
```

Comment:
I think it's better not to define these, because this ends up causing the interface to know about all the possible implementations, which runs counter to the purpose of an interface. 

In EntitySearchManager on the develop branch, there's only one reference to each of these constants. If that's still the case, I wouldn't even bother making them constants.

Reply:
That's right, my bad.

I'll add it to a Constants class specific to entity search as they are used in a couple of places.